index.js

```diff
@@ -112,6 +112,12 @@ function getWorldData(textArray, worldMode) {
 
         textLine = textArray[i]
 
+        if (!textLine.split("/")[1]) {
+            // hack shit because I'm getting weird extra lines
+            // could perhaps safely remove this by starting at i=1
+            continue

```

Comment:
To clarify: the new location I'm grabbing adventure mode data from seems to  give an additional line at the start that doesn't signify anything for this purpose. Hence, starting at `i=1` would work as an alternative to this conditional branch.

That said, I felt the safer option to prevent breakage was this conditional in the end--I can only test so many save files, and even if all the ones I tested followed this pattern I can't be certain they all will. I still don't 100% understand how that extra line manages to be scooped up every time in the first place, so I can't be certain that it couldn't happen anywhere else than the first line.